### ISSUE TITLE
intercom iOS sdk is super out of date

### ISSUE THREAD

**Issue by jpike88** [capacitor-community/intercom#112]
current version is 16... that's back from Jan 11 2024

**Comment by jcesarmobile** [comment]
Actually the plugin doesn't really specify a version, you are referring to the Podfile, but that version is only used for development, also the version is specified as `>=16`, so it install version 18.
The `.podspec` doesn't specify a version, so will also use latest version available. 
But that causes issues because the plugin supports iOS 13 and intercom 18 requires iOS 15.

**Comment by shifanpark** [comment]
Is there. any way i can pass android intercom SDK version to latest? in android build.gradle its specify 12.5.0 ,, but their latest  is 15.11.3
[15.11.3](https://github.com/intercom/intercom-android/releases/tag/15.11.3)

**Comment by jcesarmobile** [comment]
Using SDK 18 is a breaking change.
As I had already mentioned in this issue, SDK 18 requires iOS 15, while the plugin still allows iOS 13.
Apps won’t compile unless the users bump the deployment target.

**Comment by stewones** [comment]
> ... while the plugin still allows iOS 13

@jcesarmobile you say that because of the podspec pointing the deployment_target to 13? I just bumped it to 15 now

![Image](https://github.com/user-attachments/assets/c248a479-1cfd-47ee-90c9-10f35b6a3f95)

**Comment by jcesarmobile** [comment]
yeah, but those kind of changes should be done in a major release and document that the plugin requires to use deployment target 15 in user apps since Capacitor 6 supports iOS 13 by default and Capacitor 7 supports iOS 14 by default

**Comment by stewones** [comment]
> yeah, but those kind of changes should be done in a major release and document that the plugin requires to use deployment target 15 in user apps since Capacitor 6 supports iOS 13 by default and Capacitor 7 supports iOS 14 by default

Yeah that's why I bumped the plugin to v7 right?

Readme is missing this info though I agree

**Comment by stewones** [comment]
@jcesarmobile but I regretted it. do you think it's gonna be too bad if I do a capacitor 7 upgrade while still in v7? I really wanted the plugin version to match the Capacitor one.

**Comment by jcesarmobile** [comment]
technically the SDK bumps and updating it to support Capacitor 7 are both breaking changes and should have been done in 7.0.0, not on a future 7.x, but it's really up to you.

**Comment by stewones** [comment]
yeah I know, I yolo'ed that version. I'll fix this with a patch now so we can keep aligned with core. will also add info on Readme regarding the target.
